config/samples/cinder_v1beta1_cinder.yaml

```diff
@@ -4,19 +4,23 @@ metadata:
   name: cinder
   namespace: openstack
 spec:
-  databaseHostname: mariadb
-  cinderAPIReplicas: 1
-  cinderSchedulerReplicas: 1
-  cinderBackupReplicas: 1
-  cinderBackupNodeSelectorRoleName: worker
-  cinderSecret: cinder-secret
-  novaSecret: nova-secret
-  cinderAPIContainerImage: docker.io/tripleomaster/centos-binary-cinder-api:current-tripleo
-  cinderSchedulerContainerImage: docker.io/tripleomaster/centos-binary-cinder-scheduler:current-tripleo
-  cinderBackupContainerImage: docker.io/tripleomaster/centos-binary-cinder-backup:current-tripleo
+  serviceUser: cinder
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: cinder
+  cinderAPI:
+    replicas: 1
+    containerImage: docker.io/tripleomaster/centos-binary-cinder-api:current-tripleo
```

Comment:
should use images from quay.io/tripleo...

Reply:
Ack